tac/project-updates/2025/2025-annual-Hiero.md

```diff
@@ -0,0 +1,67 @@
+grand_parent: Project Updates
+---
+
+## Project Health
+
+The LFX Insights page of Hiero can be found [here](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=hiero)
+
+The Hiero project started in September 2024 and is growing since that.
+Next to the transition of all Hedera repositories that are needed to run a network we already added some repositories to the project that are community driven.
+With those projects we already onboarded new contributors to Hiero and even got some new maintainers coming from the community.
+
+Next to transitioning the repositories we are also setting up a more and more open governance model for the project.
+We now have several public meetings a week on that we discuss the project and the future of it.
+
+All repositories of the Hiero org receive regular updates and are maintained by the maintainers.
+
+## Maintainer Diversity
+
+<mark>_How many maintainers do you have, and which organizations are they from? How has the maintainers and diversity of your maintainers changed in the past year? Has the number of active maintainers increased/decreased? Has the diversity of maintainers increased/decreased? Please include a link to your existing [MAINTAINERS file](../guidelines/MAINTAINERS-guidelines.md) and the MAINTAINERS file from last year (if appropriate). This is a good opportunity to ensure that your MAINTAINERS file is up to date and to retire any maintainers._
+</mark>
+
+As a new project it is always hard to start creating a diverse community.
+We are really happy, that we already have maintainers and contributors from different companies and even some that are not employed by a company.
+This includes new contributors (and even committers and maintainers) as you can see as example in the contributor list of https://github.com/hiero-ledger/hiero-solo-action that is a community driven project which already attracting new contributors from India and Africa.
+Another positive example for contributions is the YouTube tutorial series for our Python SDK that has been created by a community member: https://www.youtube.com/playlist?list=PL6uVtTx-oqY8V-HBfenONJrsA0Z8xJymX
+
+## Project Adoption
+
+<mark>_What do you know about adoption, and how has this changed since your last review or since being accepted into LF Decentralized Trust? If you can list companies that are adopters of your project, please do so. Feel free to link to an existing ADOPTERS file if appropriate._
+</mark>
+
+## Goals
+
+### Performance Against Prior Goals
+
+Our initial main goal was to move all Hedera repositories to Hiero.
+While we are mostly done with that topic we already started to refactor the code to a more vendor neutral / unbranded state.
+Here we are currently concentrating on the SDKs and discuss workflows, roadmaps, and patterns.
+Information and current state on the transition from Hedera can be found [here](https://github.com/hiero-ledger/hiero/blob/main/transition.md).
```

Comment:
Do we also want to mention the other transition tracker for all the community driven project transition?